pysb/kappa.py

```diff
@@ -30,6 +29,99 @@
 except ImportError:
     pass
 
+# Cached value of Kappa program paths
+_kasim_path = None
+_kasa_path = None
+
+def _check_dist_dir(kappa_prog, dist_dir):
+    kappa_prog_path = os.path.join(dist_dir, kappa_prog)
+    if os.access(kappa_prog_path, os.F_OK):
+        return kappa_prog_path
+    else:
+        return None
+
+def set_kappa_path(dir):
+    """Set the path to the KaSim and KaSa executables.
+
+    Parameters
+    ----------
+    dir : string
+        Directory containing KaSim and Kasa executables.
+    """
+
+    global _kasim_path
+    global _kasa_path
+    # Check locations of both KaSim and KaSa
+    kasim_path = _check_dist_dir('KaSim', dir)
+    kasa_path = _check_dist_dir('KaSa', dir)
+
+    # Check that KaSim executable exists and is not a directory
```

Comment:
Comment is inaccurate. The check was already done above, and it doesn't check that the full path is not a directory. The "not a directory" bit can probably just be dropped entirely, anyway.